nrf_security/cmake/psa_crypto_config.cmake

```diff
@@ -147,7 +147,11 @@ kconfig_check_and_set_base_to_one(PSA_CRYPTO_DRIVER_ECC_SECP_R1_224_OBERON)
 kconfig_check_and_set_base_to_one(PSA_CRYPTO_DRIVER_ECC_SECP_R1_256_OBERON)
 
 # Convert builtin driver configurations
-kconfig_check_and_set_base_to_one(MBEDTLS_PSA_BUILTIN_CC3XX)
```

Comment:
Commit message does not mention why this was removed.
Is it correct?

Reply:
Fixed